ref/gradescope.md

```diff
@@ -74,7 +83,7 @@ Have a productive week!
 <!-- Acknowledgements
 {: .fs-4 }
 
-* Specials thanks to Liubov Kurafeeva and Roman Beltiukov for creating the initial instructions and screenshots.
+* Special thanks to Liubov Kurafeeva and Roman Beltiukov for creating the initial instructions and screenshots.
```

Comment:
@kimiaf1998 @iso-p please update the acknowledgements with the special thanks for your help! 🙏🏼

Reply:
@kimiaf1998 @ykharitonova should we open a new PR to address those changes OR there is a way of adding a new commit to this PR? I am not so sure about this part.

Reply:
I believe that the new commits might be reflected in this PR.

github also introduced the "convert to draft" functionality, which I haven't explored yet, but it sounds like it allows you to make changes to the PR...

Reply:
@iso-p I'll fix these comments. Don't worry about it.